lib/__tests__/standalone.test.mjs

```diff
@@ -466,3 +467,99 @@ describe('existing codeFilename for nested config detection', () => {
 		expect(results[0].warnings).toHaveLength(1);
 	});
 });
+
+describe('standalone with fix options for CSS syntax errors', () => {
+	const cssWithSyntaxError = '// comment';
+	const cssIncompleteRule = 'a {';
+	const cssNonSyntaxError = stripIndent`
+    a { colr: hsl(20, 10%, 30%, 5%); }
+  `;
+	const expectedFixedCode = stripIndent`
+    a { colr: hsl(20deg 10% 30% / 5%); }
+  `;
```

Comment:
[suggestion] Can you fix the indentation with whitespaces, not tabs? Unluckily, Prettier doesn't format code inside template literals.

```suggestion
	const cssNonSyntaxError = stripIndent`
		a { colr: hsl(20, 10%, 30%, 5%); }
	`;
	const expectedFixedCode = stripIndent`
		a { colr: hsl(20deg 10% 30% / 5%); }
	`;
```

Reply:
Or we can also write them with one line:

```suggestion
	const cssNonSyntaxError = 'a { colr: hsl(20, 10%, 30%, 5%); }';
	const expectedFixedCode = 'a { colr: hsl(20deg 10% 30% / 5%); }';
```